pipeline/acquisition.py

```diff
@@ -0,0 +1,375 @@
+'''
+Schema of aquisition information.
+'''
+import re
+import os
+from datetime import datetime
+
+import numpy as np
+import scipy.io as sio
+import datajoint as dj
+import h5py as h5
+
+from . import reference, subject, behavior, helper_functions
+
+schema = dj.schema(dj.config.get('database.prefix', '') + 'gi2017_acquisition')
+
+
+@schema
+class ExperimentType(dj.Lookup):
+    definition = """
+    experiment_type: varchar(64)
+    """
+    contents = [
+        ['behavior'], ['extracelluar'], ['photostim']
+    ]
+
+
+@schema 
+class ActionLocation(dj.Manual): 
+    definition = """ # Information relating the location of any experimental task (e.g. recording (extra/intra cellular), stimulation (photo or current) )
+    -> reference.BrainLocation
+    -> reference.CoordinateReference
+    coordinate_ap: decimal    # in mm, anterior positive, posterior negative 
+    coordinate_ml: decimal    # in mm, always postive, number larger when more lateral
+    coordinate_dv: decimal    # in mm, always postive, number larger when more ventral (deeper)
+    """
+    
+    
+#@schema
+#class PhotoStim(dj.Manual):
+#    definition = """
+#    photo_stim_id: int
+#    ---
+#    photo_stim_wavelength: int
+#    photo_stim_method: enum('fiber', 'laser')
+#    -> reference.BrainLocation.proj(photo_stim_location="brain_location")
+#    -> reference.Hemisphere.proj(photo_stim_hemisphere="hemisphere")
+#    -> reference.CoordinateReference.proj(photo_stim_coordinate_ref="coordinate_ref")
+#    photo_stim_coordinate_ap: float    # in mm, anterior positive, posterior negative 
+#    photo_stim_coordinate_ml: float    # in mm, always postive, number larger when more lateral
+#    photo_stim_coordinate_dv: float    # in mm, always postive, number larger when more ventral (deeper)
+#    """
+
+
+@schema
+class Session(dj.Manual):
+    definition = """
+    -> subject.Subject
+    session_time: datetime    # session time
+    ---
+    session_directory = "": varchar(256)
+    session_note = "" : varchar(256) 
+    """
+
+    class Experimenter(dj.Part):
+        definition = """
+        -> master
+        -> reference.Experimenter
+        """
+
+    class ExperimentType(dj.Part):
+        definition = """
+        -> master
+        -> ExperimentType
+        """
+
+
+@schema
+class IntracellularInfo(dj.Manual):
+    definition = """ # Table containing information relating to the intracelluar recording (e.g. cell info)
+    -> Session
+    cell_id: varchar(36) # a string identifying the cell in which this intracellular recording is concerning
```

Comment:
Similar thing here. The data type already states that this is a string so just dive into what it is: `# unique identifier of the cell`